### DISCUSSION TITLE
Question about how to use a risk-free rate higher than zero

### DISCUSSION THREAD

**Discussion by tresbagley** [dcajasn/Riskfolio-Lib#99] [Q&A] [answered]
I have tried to run the port.optimization function using a different risk-free rate higher than zero.  When I try to set rf=0.02 (to use 2%), I get the error: "The problem doesn't have a solution with actual input parameters."

I have searched through your documentation and examples and I did not see any instance of a risk-free rate being used other than the default of 0.

What is the proper syntax (or format) to use a risk-free rate other than zero?  

I have searched through your documentation and examples and I did not see any instance of a risk-free rate being used other than the default of 0.

Thanks in advance!

**Comment by dcajasn** [comment, marked as answer]
Hi @tresbagley,

You must convert the risk free rate to daily frequency, for example a 2% annual rate must be transformed to rf=0.02/252.

Best regards,
Dany

**Reply by tresbagley** [reply]
That works! Thank you so much!
